core-services/prow/02_config/_plugins.yaml

```diff
@@ -411,10 +411,6 @@ config_updater:
       cluster_groups:
       - build_farm_ci
       name: prow-job-cluster-launch-installer-openstack-upi-e2e
-    cluster/test-deploy/aws-4.0/*:
```

Comment:
If this is not being used, we should assume that it is safe to delete those files as well.

Reply:
good point